Testscripts/Linux/SetupHbKernel.sh

```diff
@@ -141,18 +152,114 @@ function Main() {
 			LogMsg "$?: Added resume=$sw_uuid in /etc/default/grub file"
 		fi
 
+		_entry=$(cat /etc/default/grub | grep 'GRUB_HIDDEN_TIMEOUT=')
+		if [ -n "$_entry" ]; then
+			sed -i -e "s/GRUB_HIDDEN_TIMEOUT=*.*/GRUB_HIDDEN_TIMEOUT=30/g" /etc/default/grub
```

Comment:
Why the `*.*`? This will match `GRUB_HIDDEN_TIMEOUTasdfaskdfjg`, too.

Reply:
Yes, I want to replace any number with 30. Actually you missed "=". In general, Azure image does not have your example.

Reply:
I didn't. It will match that: the `*` means "match 0 or more of the thing that precedes it", and the thing that precedes it is a `=`.